ush/config_defaults.yaml

```diff
@@ -2046,268 +2078,290 @@ task_get_obs_ndas:
+  TN_RUN_MET_GRIDSTAT_VX_REFC: "run_MET_GridStat_vx_REFC"
+  NNODES_RUN_MET_GRIDSTAT_VX_REFC: 1
+  PPN_RUN_MET_GRIDSTAT_VX_REFC: 1
+  MEM_RUN_MET_GRIDSTAT_VX_REFC: 2G
+  WTIME_RUN_MET_GRIDSTAT_VX_REFC: 02:00:00
+  MAXTRIES_RUN_MET_GRIDSTAT_VX_REFC: 2
 
 #----------------------------
-# VX_GRIDSTAT_24h config parameters
+# run_MET_GridStat_vx_RETOP config parameters
 #-----------------------------
-task_run_vx_gridstat_24h:
-  TN_VX_GRIDSTAT_24h: "run_gridstatvx_24h"
-  NNODES_VX_GRIDSTAT: 1
-  PPN_VX_GRIDSTAT: 1
-  MEM_VX_GRIDSTAT: 2G
-  WTIME_VX_GRIDSTAT: 02:00:00
-  MAXTRIES_VX_GRIDSTAT_24h: 2
+task_run_MET_GridStat_vx_RETOP:
+  TN_RUN_MET_GRIDSTAT_VX_RETOP: "run_MET_GridStat_vx_RETOP"
+  NNODES_RUN_MET_GRIDSTAT_VX_RETOP: 1
+  PPN_RUN_MET_GRIDSTAT_VX_RETOP: 1
+  MEM_RUN_MET_GRIDSTAT_VX_RETOP: 2G
+  WTIME_RUN_MET_GRIDSTAT_VX_RETOP: 02:00:00
+  MAXTRIES_RUN_MET_GRIDSTAT_VX_RETOP: 2
 
 #----------------------------
-# VX_POINTSTAT config parameters
+# run_MET_PointStat_vx_SFC_UPA config parameters
 #-----------------------------
-task_run_vx_pointstat:
-  TN_VX_POINTSTAT: "run_pointstatvx"
-  NNODES_VX_POINTSTAT: 1
-  PPN_VX_POINTSTAT: 1
-  MEM_VX_POINTSTAT: 2G
-  WTIME_VX_POINTSTAT: 01:00:00
-  MAXTRIES_VX_POINTSTAT: 2
+task_run_MET_PointStat_vx_SFC_UPA:
+  TN_RUN_MET_POINTSTAT_VX_SFC_UPA: "run_MET_PointStat_vx_SFC_UPA"
+  NNODES_RUN_MET_POINTSTAT_VX_SFC_UPA: 1
+  PPN_RUN_MET_POINTSTAT_VX_SFC_UPA: 1
+  MEM_RUN_MET_POINTSTAT_VX_SFC_UPA: 2G
+  WTIME_RUN_MET_POINTSTAT_VX_SFC_UPA: 01:00:00
+  MAXTRIES_RUN_MET_POINTSTAT_VX_SFC_UPA: 2
 
 #----------------------------
-# VX_ENSGRID config parameters
+# run_MET_EnsembleStat_vx_APCP01h config parameters
 #-----------------------------
-task_run_vx_ensgrid:
-  TN_VX_ENSGRID: "run_ensgridvx"
-  MAXTRIES_VX_ENSGRID: 2
-  TN_VX_ENSGRID_03h: "run_ensgridvx_03h"
-  MAXTRIES_VX_ENSGRID_03h: 2
-  TN_VX_ENSGRID_06h: "run_ensgridvx_06h"
-  MAXTRIES_VX_ENSGRID_06h: 2
-  TN_VX_ENSGRID_24h: "run_ensgridvx_24h"
-  MAXTRIES_VX_ENSGRID_24h: 2
-  TN_VX_ENSGRID_RETOP: "run_ensgridvx_retop"
-  MAXTRIES_VX_ENSGRID_RETOP: 2
-  NNODES_VX_ENSGRID: 1
-  PPN_VX_ENSGRID: 1
-  MEM_VX_ENSGRID: 2G
-  WTIME_VX_ENSGRID: 01:00:00
+task_run_MET_EnsembleStat_vx_APCP01h:
+  TN_RUN_MET_ENSEMBLESTAT_VX_APCP01H: "run_MET_EnsembleStat_vx_APCP01h"
+  NNODES_RUN_MET_ENSEMBLESTAT_VX_APCP01H: 1
+  PPN_RUN_MET_ENSEMBLESTAT_VX_APCP01H: 1
+  MEM_RUN_MET_ENSEMBLESTAT_VX_APCP01H: 2G
+  WTIME_RUN_MET_ENSEMBLESTAT_VX_APCP01H: 01:00:00
+  MAXTRIES_RUN_MET_ENSEMBLESTAT_VX_APCP01H: 2
 
 #----------------------------
-# VX_ENSGRID_REFC config parameters
+# run_MET_EnsembleStat_vx_APCP03h config parameters
 #-----------------------------
-task_run_vx_ensgrid_refc:
-  TN_VX_ENSGRID_REFC: "run_ensgridvx_refc"
-  NNODES_VX_ENSGRID: 1
-  PPN_VX_ENSGRID: 1
-  MEM_VX_ENSGRID: 2G
-  WTIME_VX_ENSGRID: 01:00:00
-  MAXTRIES_VX_ENSGRID_REFC: 2
+task_run_MET_EnsembleStat_vx_APCP03h:
+  TN_RUN_MET_ENSEMBLESTAT_VX_APCP03H: "run_MET_EnsembleStat_vx_APCP03h"
```

Comment:
The name of the task should probably change too just for consistency.

Reply:
@danielabdi-noaa Since there is no NCO requirement on the ROCOTO task names, I decided to keep them as a mix of upper and lower case in order to match the names of the log files as described in Issue #618 (and since to me this naming convention is easier to read, which is my reason to use it in the first place).  I will ask in that issue whether people have a preference on this.